runtime/executor/src/main/java/edu/snu/nemo/runtime/executor/task/VertexHarness.java

```diff
@@ -111,10 +114,17 @@ OutputCollectorImpl getOutputCollector() {
   }
 
   /**
-   * @return OutputWriters of this irVertex. (empty if none exists)
+   * @return OutputWriters for main outputs of this irVertex. (empty if none exists)
```

Comment:
What is the difference of "main" output and "additional" output? Please describe it.

Reply:
The main outputs are output with main TupleTag and additional outputs are outputs with additional TupleTags. These are from Beam programming model.

Reply:
If so, please add comments about it to this method and the method for additional output also. Thanks for the explanation!